lib/osf-components/addon/components/registries/schema-block-group-renderer/component.ts

```diff
@@ -0,0 +1,51 @@
+import { tagName } from '@ember-decorators/component';
+import { computed } from '@ember-decorators/object';
+import { alias } from '@ember-decorators/object/computed';
+import Component from '@ember/component';
+import { assert } from '@ember/debug';
+import Changeset from 'ember-changeset';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import NodeModel from 'ember-osf-web/models/node';
+import { SchemaBlock, SchemaBlockGroup } from 'ember-osf-web/packages/registration-schema';
+import defaultTo from 'ember-osf-web/utils/default-to';
+import { uniqueId } from 'osf-components/helpers/unique-id';
+import template from './template';
+
+@layout(template)
+@tagName('')
+export default class SchemaBlockGroupRenderer extends Component {
+    // Required parameters
+    schemaBlockGroup!: SchemaBlockGroup;
+    changeset!: Changeset;
+    node!: NodeModel;
+
+    // Optional params
+    disabled: boolean = defaultTo(this.disabled, false);
+    shouldShowMessages: boolean = defaultTo(this.shouldShowMessages, true);
+
+    @alias('schemaBlockGroup.optionBlocks')
+    optionBlocks!: SchemaBlock[];
+
+    didReceiveAttrs() {
+        assert('A schema group is required to render schema groups', Boolean(this.schemaBlockGroup));
+        assert('A changeset is required to render schema groups', Boolean(this.changeset));
```

Comment:
(similar to my other comment) when updating to the strategy pattern, look for args that are only used to pass through to the block renderer -- those should be passed to the strategy component instead. I think `@changeset` and `@node` here are examples.